brave/browser-laptop
# Refactor notificationBar & updateBar

Refactor notificationBar & updateBar (including the ledger notification bar which was implemented by @ayumi with PR #3856 of the issue #3257) with aphrodite.

From:
<img width="1040" alt="screenshot 2017-03-23 14 39 49" src="https://cloud.githubusercontent.com/assets/3362943/24235886/970edea0-0fe2-11e7-8713-2e109a4c108a.png">

To:
<img width="1077" alt="screenshot 2017-03-23 16 30 08" src="https://cloud.githubusercontent.com/assets/3362943/24236653/1fc48706-0fe6-11e7-91e2-0d937da0e7e9.png">

Closes #7853

Auditors:

Test Plan:
1. Close the browser
2. Edit `<UpdateNotAvailable>` to `<UpdateAvailable>` on updateBar.js
3. Edit ledger-state.json to display the ledger notification bar
4. Open the browser
5. Check for update...
6. Open https://browserleaks.com/geo
7. Wait until the ledger notification bar appears


Test Plan:
